templates/openstack/managed.html

```diff
@@ -1,725 +1,619 @@
+              updates, bug fixes, etc.</p>
+          </div>
+          <div class="col-4 u-align--center u-vertically-center u-hide--small">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/e9d3cf66-We+support+your+operations.svg",
+                alt="",
+                height="134",
+                width="134",
+                hi_def=True,
+                loading="lazy",
+              ) | safe
+            }}
+          </div>
+        </div>
+      </li>
+      <li class="p-stepped-list__item">
+        <div class="row">
+          <div class="col-8">
+            <h3 class="p-stepped-list__title">
+              Transfer
+            </h3>
+            <p>On request, we transfer control over your cloud back to your organisation. We
+              train your personnel, make sure you are able to operate the cloud yourself moving forward and perform a formal
+              handover process. We continue providing commercial support to make sure your cloud receives security updates
+              and you receive help if needed.</p>
+          </div>
+          <div class="col-4 u-align--center u-hide--small">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/2131b805-We+transfer+control.svg",
+                alt="",
+                height="134",
+                width="134",
+                hi_def=True,
+                loading="lazy",
+              ) | safe
+            }}
+          </div>
+        </div>
+      </li>
+    </ol>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Managed OpenStack pricing</h2>
+  </div>
+  <div class="row">
+    <div class="col-6 p-card">
+      <h3 class="p-heading--4">$15 per server per day</h3>
+      <hr class="u-sv1" />
+      <p>What's included:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Logging, monitoring and alerting</li>
+        <li class="p-list__item is-ticked">24x7 operations with fully staffed NOC</li>
+        <li class="p-list__item is-ticked">Proactive cloud management</li>
+        <li class="p-list__item is-ticked">Incident and problem resolution</li>
+        <li class="p-list__item is-ticked">Software updates and OpenStack upgrades</li>
+        <li class="p-list__item is-ticked">SLA and regular independent audits</li>
+        <li class="p-list__item is-ticked">Most cost-effective approach up to 300 nodes</li>
+        <li class="p-list__item is-ticked">Optional Managed Kubernetes service on top</li>
+        <li class="p-list__item is-ticked">EU and US regulatory compliance options</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Modernise your infrastructure with Managed OpenStack</h2>
   </div>
+  <div class="row">
+    <div class="col-8">
```

Comment:
Sorry, I think the heading should be within the 8 col, it looks a little odd with the title hanging over the image like that:

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>Modernise your infrastructure with Managed OpenStack</h2>
```

Reply:
Thanks Scott!